### ISSUE TITLE
2FA for ProudCity login

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1709]

**Comment by lukefretwell** [comment]
@kevindherman here's the documentation for setting it up: https://auth0.com/docs/secure/multi-factor-authentication/enable-mfa

**Comment by kevindherman** [comment]
We may need to bump up out of our grandfathered pricing but I think it would be worth it if that's the case.

**Comment by jlyon** [comment]
Yep, it looks like at a minimum we would need to upgrade to the Professional plan, which is $240/mo (versus the $13/mo we currently pay): https://manage.auth0.com/dashboard/us/proudcity/tenant/billing/subscription. That would get us support for apps like Google Authenticator and 1Password. To support SMS codes, you need to be on plan to get Enterprise MFA, which is a "contact us" situation. @kevindherman @lukefretwell

**Comment by lukefretwell** [comment]
How many users do we have? Looks like the 240 is for 1,000 users. I think we're closer to that now. Given that it's based on users we need to figure out a plan to offboard/deactivate users who aren't using the platform.

I think it's fine to go with 'Professional.' SMS is great but I think there are questions about how secure it is relative to an authenticator and it's also probably going to be cost-prohibitive.

**Comment by kevindherman** [comment]
We should definitely look into offboarding deactivated users. 

I'm not sure this will save money in this case because it seems like the 1000 users is the ceiling for the essentials plan and the floor for the pricing toggle for professional. 

The pricing  just about doubles once you hit 2500 users so having a cleanup process is worthwhile in the long term.

**Comment by jlyon** [comment]
we're currently under 1000 users, so usage hasn't been an issue.

**Comment by kevindherman** [comment]
@lukefretwell found this post that the users are actually "active users" which we're way below: https://community.auth0.com/t/trying-to-understand-mau-monthly-active-users-pricing/49222

It really just comes down to upgrading to Pro MFA which I think we need to do.

cc @jlyon
